### PR TITLE
Fix select error, fix multiple select

### PR DESCRIPTION
**Pull request by borisenkoandrei**

### CODE REVIEW

File: webpack/dev.config.js [hunk 1]
```diff
@@ -30,7 +30,16 @@ module.exports = {
         contentBase: path.resolve(__dirname, '../.tmp'),
         watchContentBase: true,
         port: 9000,
-        historyApiFallback: true
+        historyApiFallback: true,
+        headers: {
+            'Access-Control-Allow-Origin': '*',
+        },
+        proxy: {
+            '/api': {
```
Comment (ditkachuk) [inline webpack/dev.config.js]: seems demo should work without proxy, need to have option on set domain in all form requests

Comment (ditkachuk) [reply webpack/dev.config.js]: lets separate from select fixes

File: src/components/formComponents/Select.js [hunk 2]
```diff
@@ -27,11 +27,20 @@ class Select extends Component {
...
+            return undefined;
+        }
+    }
+
+    onChange = (data) => {
         const { settings, onChange } = this.props;
         const multiple = path(['multiple'], settings);
+        const value = multiple ? this.getMultipleValue(data) : data.value;
 
-        onChange(multiple ? (value && value.length ? value : null) : value);
```
Comment (ditkachuk) [inline src/components/formComponents/Select.js]: seems miss case with empty array value

File: src/components/formComponents/Select.js [hunk 3]
```diff
@@ -27,11 +27,20 @@ class Select extends Component {
...
-    onChange = ({ value }) => {
+    getMultipleValue = (values) => {
+        if (values) {
+            return values.map(({ value }) => value);
+        } else {
+            return undefined;
+        }
+    }
+
+    onChange = (data) => {
```
Comment (ditkachuk) [inline src/components/formComponents/Select.js]: event?

File: demo/components/AppForm.js [hunk 4]
```diff
@@ -29,6 +29,7 @@ class AppForm extends Component {
                         onError={this.onError}>
                         { mutation =>
                             <Form
+                                dictionaryUrl='https://jobot.dev.experium.net/api/dictionary'
```
Comment (Panacee) [inline demo/components/AppForm.js]: proxy?

Comment (Panacee) [inline demo/components/AppForm.js]: base url in constants
add props for file urls

**Pull request merged**